modules/dataquery/jsx/components/expansionpanels.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useState} from 'react';
+
+/**
+ * Component to render a single panel
+ *
+ * @param {object} props - The React Props
+ * @param {boolean} props.defaultOpen - Whether the default should default open
+ * @param {boolean} props.alwaysOpen - Whether the panel can not be collapsed
+ * @param {string} props.title - The panel title
+ * @param {React.ReactElement} props.content - The panel body
+ * @returns {React.ReactElement} - The rendered panel
+ */
+const Panel = (props: {
+    defaultOpen: boolean,
+    alwaysOpen: boolean,
+    title: string,
+    content: React.ReactElement,
+}) => {
+  const [active, setActive] = useState(props.defaultOpen);
+
+  const styles = {
+    accordion: {
+      default: {
+        width: '100%',
+        padding: '18px',
```

Comment:
The "ExpansionPanels" came from the old DQT. Instead of trying to bring the coding style of it up to date, I updated it to use `jsx/Panel` for consistency with the rest of LORIS.